action.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish TechDocs
+description: Generate and publish TechDocs
+inputs:
+  publisher:
+    description: awsS3 | googleGcs | azureBlobStorage
+    required: true
+  credentials:
+    description: 
+      Credentials required to authenticate with your publisher through the techdocs-cli
+      For GCS, provide the JSON keyfile as a base64-encoded string in GOOGLE_APPLICATION_CREDENTIALS 
```

Comment:
would it be worth being more precise that the JSON contains service account key or is `keyfile` clear enough? 🤔

Reply:
👍 if you think it would be clearer. It's slightly complicated in that I _think_ you could put a JSON token for a user in here, as well as a service account, but you shouldn't do that really

Reply:
hmm in [here](https://cloud.google.com/docs/authentication/getting-started#setting_the_environment_variable) they only mention the SA key but maybe that's just a best practice as you said